x/liquidstakeibc/types/msgs.go

```diff
@@ -282,6 +282,25 @@ func (m *MsgUpdateHostChain) ValidateBasic() error {
 			if fee.LT(sdk.ZeroDec()) || fee.GT(sdk.OneDec()) {
 				return sdkerrors.ErrInvalidRequest.Wrapf("invalid unstake fee value should be 0 <= fee <= 1")
 			}
+		case KeyLSMValidatorCap:
+			validatorCap, err := sdk.NewDecFromStr(update.Value)
+			if err != nil {
+				return fmt.Errorf("unable to parse string to sdk.Dec: %w", err)
+			}
+
+			if validatorCap.LT(sdk.ZeroDec()) || validatorCap.GT(sdk.OneDec()) {
+				return sdkerrors.ErrInvalidRequest.Wrapf("invalid validator cap value should be 0 <= cap <= 1")
+			}
+		case KeyLSMBondFactor:
+			bondFactor, err := sdk.NewDecFromStr(update.Value)
+			if err != nil {
+				return fmt.Errorf("unable to parse string to sdk.Dec: %w", err)
+			}
+
+			// -1 is the default bond factor value
+			if bondFactor.LT(sdk.NewDec(-1)) {
```

Comment:
is -0.5 allowed? I believe we want to use -1 as a default ( infinite ) if possible lets add -1 or GTE(0), if zero is allowed

Reply:
-1 is the default value which means the functionality is not "active", hence we should be able to LS if that is the case

https://github.com/cosmos/cosmos-sdk/blob/0af2f4da004cbea6414a8bad56e8bdcd45badf1e/x/staking/types/params.go#L40